webapp/src/js/controllers/reports.js

```diff
@@ -448,7 +450,7 @@ angular
     var syncCheckboxes = function() {
```

Comment:
I personal would love to see us using `const` and `let` in this redux work? var stinks

Reply:
Agreed! I'm certainly using `const` and `let` whenever I add anything new but haven't been converting old `var`s mainly for the sake of keeping the number of lines changed down in the PR (so it's only the relevant stuff being reviewed)